app/components/content/talk_to_us_component.html.erb

```diff
@@ -25,6 +23,12 @@
           <%= render ChatComponent.new %>
         </div>
       </div>
+        </div>
+      <div class="inset">
+          <%= render Content::InsetTextComponent.new(
+            header: "Non-UK citizens:",
+            text: "You can call us on <a href=\"tel:+448003892500\">+44 800 389 2500</a>, or use the free live chat service. Calls will be charged at your country's standard rate.", color: "purple-white"
```

Comment:
@emmaachesongray I'm not sure why `text` and `color` are on the same line now - they should really be on separate lines

However, I know you had linting issues around this so if you want to leave and get Jen to have a look when she considers the purple/white background issue when back from leave, that's fine with me as it looks OK on the front-end in the meantime